### PR TITLE
[fx] test tracer on diffuser modules.

### PR DESCRIPTION
**Pull request by super-dainiu**
### Updates
I tested diffuser modules (i.e. Unet, CLIP, and VAE) on our `ColoTracer`. Sadly, we cannot trace unet currently. But the good news is that VAE and CLIP are able to trace.

### Test
image

### CODE REVIEW

File: tests/test_fx/test_tracer/test_hf_model/test_hf_albert.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import transformers
-import torch
 import pytest
+import torch
+import transformers
 from utils import trace_model_and_compare_output
 
 BATCH_SIZE = 2
-SEQ_LENGHT = 16
+SEQ_LENGTH = 16
```
Comment (super-dainiu) [inline tests/test_fx/test_tracer/test_hf_model/test_hf_albert.py:7]: fix typo

File: tests/test_fx/test_tracer/test_hf_model/test_hf_albert.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import transformers
-import torch
 import pytest
+import torch
+import transformers
```
Comment (super-dainiu) [inline tests/test_fx/test_tracer/test_hf_model/test_hf_albert.py:3]: These are forced by the latest `pre-commit` hook.

File: tests/test_fx/test_tracer/test_hf_model/test_hf_diffuser.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+import transformers
+from torch.fx import GraphModule
+from utils import trace_model_and_compare_output
+
+from colossalai.fx import ColoTracer
+
+BATCH_SIZE = 2
+SEQ_LENGTH = 5
+HEIGHT = 224
+WIDTH = 224
```
[the remark this answers is not in the document]
Comment (FrankLeeeee) [reply tests/test_fx/test_tracer/test_hf_model/test_hf_diffuser.py:13]: so?

**Review by FrankLeeeee — APPROVED** [review]

File: tests/test_fx/test_tracer/test_hf_model/test_hf_diffuser.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (super-dainiu) [reply tests/test_fx/test_tracer/test_hf_model/test_hf_diffuser.py:13]: so the input has to be 224 that large

[the remark this answers is not in the document]
Comment (FrankLeeeee) [reply tests/test_fx/test_tracer/test_hf_model/test_hf_diffuser.py:13]: ok.

**Pull request merged**